### ISSUE TITLE
BUG     The program does not report an error, but it will not be executed

### ISSUE THREAD

**Issue by StochasticGame** [Anonymous1563/Bayesian-Enhancement-Model#18]
At runtime, the code often stops at this line of code, but the program does not report an error, it just does not move, and this problem is only found during debugging

`python Enhancement/eval.py --opt /data/code/Bayesian-Enhancement-Model/experiments/CG_UNet_UIEB/CG_UNet_UIEB.yml --weights /data/code/Bayesian-Enhancement-Model/experiments/CG_UNet_UIEB/best_psnr_20.87_282000.pth --cond_opt /data/code/Bayesian-Enhancement-Model/experiments/IE_UNet_UIEB/IE_UNet_UIEB.yml --cond_weights /data/code/Bayesian-Enhancement-Model/experiments/IE_UNet_UIEB/best_psnr_34.37_141000.pth --dataset DICM --input_dir /data/data/TURBID_3D/test_data/input/ --no_ref uiqm_uciqe --uiqm_weight 1.0`

![Image](https://github.com/user-attachments/assets/e6383582-1a1d-49c6-81bd-425691c3b50a)

`one_preds.append(cond_net(torch.cat([input_expended, sub_one_pred_conds], dim=1))[-1].cpu())`

**Comment by BinCVER** [comment]
What GPU do you use? Maybe setting `--num_samples 25` or a smaller number can solve your problem.

If you found our repository useful, please consider giving it a star, as it means a lot to us. Thank you!
